packages/expo-modules-core/ios/ModuleRegistryAdapter/EXViewManagerAdapterClassesRegistry.h

```diff
@@ -12,5 +12,6 @@
 @interface EXViewManagerAdapterClassesRegistry : NSObject
 
 - (Class)viewManagerAdapterClassForViewManager:(EXViewManager *)viewManager;
++ (Class)createViewManagerAdapterClassForViewManager:(EXViewManager *)viewManager;
```

Comment:
This is almost the same as the method above, but static and doesn't cache results.